Tiqr/Classes/AuthenticationSummaryViewController.m

```diff
@@ -44,15 +45,17 @@ @interface AuthenticationSummaryViewController ()
 @property (nonatomic, strong) IBOutlet UILabel *serviceProviderDisplayNameLabel;
 @property (nonatomic, strong) IBOutlet UILabel *serviceProviderIdentifierLabel;
 @property (nonatomic, strong) IBOutlet UIButton *returnButton;
+@property (nonatomic, copy) NSString *PIN;
 
 @end
 
 @implementation AuthenticationSummaryViewController
 
-- (instancetype)initWithAuthenticationChallenge:(AuthenticationChallenge *)challenge {
+- (instancetype)initWithUsedPIN:(NSString *)PIN {
     self = [super initWithNibName:@"AuthenticationSummaryView" bundle:nil];
 	if (self != nil) {
-		self.challenge = challenge;
+		self.challenge = ServiceContainer.sharedInstance.challengeService.currentAuthenticationChallenge;
+        self.PIN = PIN;
```

Comment:
Putting the PIN in a member might mean it gets kept around in memory longer than needed, potentially making it vulnerable to an attacker looking at the app's memory. Is this avoidable?